src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const stringSplitted = sourceString.split(';');
+
+  const filtered = stringSplitted.filter((item) => item.trim().length);
+
+  const dividedWords = filtered.map((words) => words.split(':'));
```

Comment:
[CODE STYLE] - The name `dividedWords` is a bit misleading here. You're not dividing words, but rather splitting style declarations into keys and values. Consider renaming this variable to `splitDeclarations` or `keyValuePairs`.